test/component_failures_test.py

```diff
@@ -186,5 +186,17 @@ def testPlasmaStoreFailed(self):
 
     ray.worker.cleanup()
 
+  def testDriverLives(self):
+    ray.worker.init()
+    for process in [
+        ray.services.all_processes[ray.services.PROCESS_TYPE_PLASMA_STORE][0],
+        ray.services.all_processes[ray.services.PROCESS_TYPE_PLASMA_MANAGER][0],
+        ray.services.all_processes[ray.services.PROCESS_TYPE_LOCAL_SCHEDULER][0],
+        ray.services.all_processes[ray.services.PROCESS_TYPE_GLOBAL_SCHEDULER][0],
+        ]:
+      process.terminate()
+      process.wait()
```

Comment:
Want to add another test does the same thing as this, but does all of the `process.terminate()` calls and then all of the `process.wait()` calls instead of serializing them?